test/packages/with-custom-agent/auditd_manager_independent_agent/data_stream/auditd/_dev/test/system/test-default-config.yml

```diff
@@ -0,0 +1,12 @@
+data_stream:
+  vars:
+    audit_rules:
+      - "-a always,exit -F arch=b64 -S execve,execveat -k exec"
+    preserve_original_event: true
+agent:
+  user: "root"
+  # shared_pid_address_space: true ?
+  pid_mode: "host"
+  capabilities:
+    - AUDIT_CONTROL
+    - AUDIT_READ
```

Comment:
Currently, this file allows additional properties. However, it probably would be better to at least add `agent` field as an object in this file. WDYT ?

Reply:
Yes please.

Reply:
Created PR for this : https://github.com/elastic/package-spec/pull/739